File: projects/calculator/cpp/calculator.cpp
#include <iostream>
using namespace std;

int main() {
    cout << "Simple Calculator\n";
    cout << "1. Add  2. Subtract  3. Multiply  4. Divide\n";
    cout << "Choice: ";
    int choice;
    cin >> choice;

    double a, b;
    cout << "First number: ";
    cin >> a;
    cout << "Second number: ";
    cin >> b;

    switch (choice) {
        case 1: cout << "Result: " << a + b << endl; break;
        case 2: cout << "Result: " << a - b << endl; break;
        case 3: cout << "Result: " << a * b << endl; break;
        case 4:
            if (b != 0) cout << "Result: " << a / b << endl;
            else cout << "Cannot divide by zero" << endl;
            break;
        default: cout << "Invalid" << endl;
    }
    return 0;
}

File: projects/calculator/go/calculator.go
package main

import "fmt"

func main() {
	fmt.Println("Simple Calculator")
	fmt.Println("1. Add")
	fmt.Println("2. Subtract")
	fmt.Println("3. Multiply")
	fmt.Println("4. Divide")

	var choice int
	fmt.Print("Enter choice (1-4): ")
	fmt.Scanln(&choice)

	var a, b float64
	fmt.Print("First number: ")
	fmt.Scanln(&a)
	fmt.Print("Second number: ")
	fmt.Scanln(&b)

	switch choice {
	case 1:
		fmt.Printf("Result: %v\n", a+b)
	case 2:
		fmt.Printf("Result: %v\n", a-b)
	case 3:
		fmt.Printf("Result: %v\n", a*b)
	case 4:
		if b != 0 {
			fmt.Printf("Result: %v\n", a/b)
		} else {
			fmt.Println("Cannot divide by zero")
		}
	default:
		fmt.Println("Invalid choice")
	}
}
